.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,48 @@
+name: postman-pull-request
+on: [push, pull_request] // this action runs on each new push 
+                         // and pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+        postgresql-version: [11]
```

Comment:
Can we use postgresql version 12? The server image is `postgres:12.3`.

Reply:
Yup, I'll change that